achyut/src/main/java/com/lftechnology/com/programs/PalindromeProgram.java

```diff
@@ -0,0 +1,52 @@
+package com.lftechnology.com.programs;
+
+import java.util.Scanner;
+
+public class PalindromeProgram {
+
+    private String number;
+    private Scanner scanner;
+
+    public PalindromeProgram() {
+
+    }
+
+    public void run() {
+        this.initiateRun();
+        System.out.println("Please Enter the Number of Multiplier");
+        scanner = new Scanner(System.in);
+        this.number = scanner.nextLine();
+        Boolean isPalindrome = this.isPalindrome();
+        if (isPalindrome == true) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the literal "true" boolean value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar.java.lftechnology.com:9000/coding_rules#rule_key=squid%3AS1125)